store/sqlstore/post_store.go

```diff
@@ -2840,8 +2840,16 @@ func (s *SqlPostStore) updateThreadAfterReplyDeletion(transaction *sqlxTxWrapper
 			}
 		}
 
+		lastReplyAtSubquery := sq.Select("COALESCE(MAX(CreateAt), 0)").
+			From("Posts").
+			Where(sq.Eq{
+				"RootId":   rootId,
+				"DeleteAt": 0,
+			})
+
 		updateQueryString, updateArgs, err := updateQuery.
 			Set("ReplyCount", sq.Expr("ReplyCount - 1")).
+			Set("LastReplyAt", lastReplyAtSubquery).
```

Comment:
Out of scope for this PR, but given the above subquery has to read all the affected rows anyway, would it make sense to throw a `COUNT(*)` into the subquery to explicitly recount `ReplyCount` as well vs. the `ReplyCount--` semantics which seem potentially brittle?

Reply:
@lieut-data that would make sense, yes. But we can't just throw it in the subquery, it seems MySQL doesn't allow it.
Should I just query for the values separately?

Reply:
For simplicity, I think it would be fine to just do a second subquery. But I'm also fine with deferring this entirely out of this PR so as to reduce churn -- mostly just thinking about next steps :)